h2o-algos/src/main/java/hex/StackedEnsembleModel.java

```diff
@@ -343,43 +347,50 @@ public void checkAndInheritModelProperties() {
         // NOTE: if we loosen this restriction and fold_column is set add a check below.
         Frame aTrainingFrame = aModel._parms.train();
         if (trainingFrameRows != aTrainingFrame.numRows() && !this._parms._is_cv_model)
-          throw new H2OIllegalArgumentException("Base models are inconsistent: they use different size(number of rows) training frames.  Found number of rows: " + trainingFrameRows + " and: " + aTrainingFrame.numRows() + ".");
+          throw new H2OIllegalArgumentException("Base models are inconsistent: they use different size (number of rows) training frames." +
+              " Found: "+trainingFrameRows+" (StackedEnsemble) and "+aTrainingFrame.numRows()+" (model "+k+").");
 
         if (! responseColumn.equals(aModel._parms._response_column))
-          throw new H2OIllegalArgumentException("Base models are inconsistent: they use different response columns.  Found: " + responseColumn + " and: " + aModel._parms._response_column + ".");
+          throw new H2OIllegalArgumentException("Base models are inconsistent: they use different response columns." +
+              " Found: "+responseColumn+"(StackedEnsemble) and "+aModel._parms._response_column+" (model "+k+").");
+        
+//        if (blending_mode && _parms._blending.equals(aModel._parms._train)) {
+//          throw new H2OIllegalArgumentException("Base model `"+k+"` was trained with the StackedEnsemble blending frame.");
+//        }
 
         // TODO: we currently require xval; loosen this iff we add a separate holdout dataset for the ensemble
-
```

Comment:
this block has only changes in indentation due to inclusion in `if (cv_required_on_base_model)` condition